Dockerfile

```diff
@@ -0,0 +1,51 @@
+# ===============================================
+FROM helsinkitest/node:12-slim as appbase
+# ===============================================
+# Offical image has npm log verbosity as info. More info - https://github.com/nodejs/docker-node#verbosity
+ENV NPM_CONFIG_LOGLEVEL warn
+
+# We will use the `node` user which comes with the official node alpine image
+ENV HOME /home/node
+
+# set our node environment, either development or production
+# defaults to production, compose overrides this to development on build and run
+ARG NODE_ENV=production
+ENV NODE_ENV $NODE_ENV
+
+# Global npm deps in a non-root user directory
+ENV NPM_CONFIG_PREFIX=/home/node/.npm-global
+ENV PATH=$PATH:/home/node/.npm-global/bin
+
+# Yarn
+ENV YARN_VERSION 1.19.1
+RUN yarn policies set-version $YARN_VERSION
+
+# UID and GID mapping
+RUN groupmod -g 999 node && usermod -u 999 -g 999 node
+
+# Create a node_modules directory and set it's permissions
+RUN mkdir -p $HOME/app/node_modules && chown -R node:node $HOME/app
```

Comment:
I have not seen this as being necessary at any point creating the node_modules dir like this. It will be created by  the same user as the one that runs `yarn` anyway, which should be the `appuser`

The CHOWN is not needed as you should be using the `appuser`

Reply:
check, I have removed this.